recipes/pdal/all/test_package/conanfile.py

```diff
@@ -0,0 +1,17 @@
+import os
+
+from conans import ConanFile, CMake, tools
+
+class TestPackageConan(ConanFile):
+    settings = "os", "compiler", "build_type", "arch"
+    generators = "cmake", "cmake_find_package_multi"
```

Comment:
`cmake_find_package_multi` currently is not used (incase the other suggestions are not applied)

Reply:
Agree, it depends on the test approach. For the current version it is needed.